integration-tests/tests/hacbs/add-component-to-application-via-hacbs-tab.spec.ts

```diff
@@ -0,0 +1,116 @@
+import { AddComponentPage } from "../../support/pages/AddComponentPage";
+import { ApplicationDetailPage } from "../../support/pages/ApplicationDetailPage";
+import { ComponentPage } from "../../support/pages/ComponentsPage";
+import { OverviewTabPage } from "../../support/pages/hacbs/OverviewTabPage";
+import { Applications } from "../../utils/Applications";
+import { Common } from "../../utils/Common";
+import { ComponentsTabPage } from "../../support/pages/hacbs/ComponentsTabPage";
+import { actionsDropdown } from "../../support/pageObjects/hacbs-po";
+
+
+describe('', () => {
+    const applicationName = Common.generateAppName();
+    const addComponent = new AddComponentPage();
+    const componentPage = new ComponentPage();
+    const overviewTabPage = new OverviewTabPage();
+    const componentsTabPage = new ComponentsTabPage();
+    const applicationDetailPage = new ApplicationDetailPage();
+    const publicRepoOfJava = 'https://github.com/dheerajodha/devfile-sample-code-with-quarkus';
+    const publicRepoOfGo = 'https://github.com/devfile-samples/devfile-sample-go-basic.git';
+    const publicRepoOfNodeJS = 'https://github.com/nodeshift-starters/devfile-sample.git';
+    const componentOfJava = 'java-quarkus';
+    const componentOfGo = 'go';
+    const componentOfNodeJS = 'nodejs'
+
+    before(function () {
+        // Enable HACBS
+        Common.enableHACBS();
+    });
+
+    after(function () {
+        //Open components page and delete all 3 components
+        applicationDetailPage.deleteComponent(componentOfJava);
+        applicationDetailPage.createdComponentNotExists(componentOfJava);
+
+        applicationDetailPage.deleteComponent(componentOfGo);
+        applicationDetailPage.createdComponentNotExists(componentOfGo);
+
+        applicationDetailPage.deleteComponent(componentOfNodeJS);
+        applicationDetailPage.createdComponentNotExists(componentOfNodeJS);
```

Comment:
this would look a lot better using a loop, even if you stuck the component names in a list and iterated using `for.. of`

or if you are just looking to clean up after the tests, deleting the application would suffice

Reply:
Just deleting the application directly sounds so much better. Updated.